index.js

```diff
@@ -77,15 +79,25 @@ function storeOptionValue(parseOptions, option, value, result) {
   }
 }
 
-const parseArgs = (
+const parseArgs = ({
   argv = getMainArgs(),
   options = {}
-) => {
+} = {}) => {
   validateArray(argv, 'argv');
   validateObject(options, 'options');
-  for (const key of ['withValue', 'multiples']) {
-    if (ObjectHasOwn(options, key)) {
-      validateArray(options[key], `options.${key}`);
+  for (const [option, optionConfig] of ObjectEntries(options)) {
```

Comment:
for..of is never robust, because of Symbol.iterator; this should use `ArrayPrototypeForEach(ObjectEntries(options), ([option, optionConfig]) => {` instead.

Reply:
Thanks! https://github.com/pkgjs/parseargs/pull/63/commits/ecf1fb09a855ccef0f4a4eaba410edb9e90610ed